traits/etsconfig/tests/test_etsconfig.py

```diff
@@ -19,6 +19,7 @@
 import time
 
 import unittest
+from unittest.mock import patch
```

Comment:
Could we use `import unittest.mock` instead of the `from ... import` here? It's not a big deal, but it's the style used elsewhere in the repository (and it's the one that I personally prefer).

Reply:
Also, let's lose the extra space between the `time` import and the `unittest` import while we're here.

Reply:
Half-done: I didn't change the import; just the import spacing.